.expeditor/verify.pipeline.yml

```diff
@@ -1 +1,18 @@
-# nothing yet, but stay tuned
\ No newline at end of file
+---
+expeditor:
+  defaults:
+    buildkite:
+      retry:
+        automatic:
+          limit: 1
+      timeout_in_minutes: 30
+
+steps:
+
+- label: ":windows: :ruby: ruby-plus-devkit plan"
+  commands:
+    - bin/ci/verify_a_plan.ps1 -Plan ruby-plus-devkit
+  expeditor:
+    executor:
+      windows:
+        os-version: 2016
```

Comment:
Why 2016?

Reply:
Because of the cult I cargoed. 

![tenor-42740745](https://user-images.githubusercontent.com/517302/66660999-312c5200-ec14-11e9-929f-edb7debf2ec2.gif)

I'm game to try something else. Especially since there's a smoke test for the package.

Reply:
I would recommend removing this value and defaulting to latest (which currently is 2019).